scripts/update-manpages

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# =============
+# = Functions =
+# =============
+
+cleanup() {
+        printf '👋  Change back to original directory “%s”\n' "$CURRENT_DIR"
+        cd "$CURRENT_DIR" || exit
+}
+
+# ========
+# = Init =
+# ========
+
+CURRENT_DIR="$PWD"
+SCRIPTS_DIR=$(dirname "$0")
+. "$SCRIPTS_DIR/realpath"
+
+ELEKTRA_DIR=`realpath "$SCRIPTS_DIR"/..`
+BUILD_DIR="$ELEKTRA_DIR/build"
+MANPAGES_INPUT_DIR="$BUILD_DIR/doc/help"
+MANPAGES_OUTPUT_DIR="$ELEKTRA_DIR/doc/man"
+
+command -v ronn >/dev/null 2>&1 || { printf 'Unable to locate ronn' >&2; exit 1; }
+command -v make >/dev/null 2>&1 || { printf 'Unable to locate Make' >&2; exit 1; }
+command -v cmake >/dev/null 2>&1 || { printf 'Unable to locate CMake' >&2; exit 1; }
+
+mkdir -p "$BUILD_DIR" || { printf 'Unable to create “%s”' "$BUILD_DIR" >&2; exit 1; }
+
+# ========
+# = Main =
+# ========
+
+printf '👋  Change into build directory “%s”\n' "$BUILD_DIR"
+cd "$BUILD_DIR" || exit
```

Comment:
Is it necessary to start a fresh build here? What about splitting the configure phase and the copying phase into two scripts?

Reply:
> Is it necessary to start a fresh build here?

Not really. The script now checks the CMake cache and does not regenerate the build system if the option `BUILD_DOCUMENTATION` is enabled.